wiki/Modifying-Your-Game-with-Scripts.md

```diff
@@ -98,10 +130,58 @@ Hooks.once("init", () => {
 });
 ```
 
-## Changing the Max Level Cap
+## Change the Max Level Cap
 ```js
 /// Increases the Maximum Level for a Player Actor to 30
 Hooks.once("init", () => {
   CONFIG.DND5E.maxLevel = 30;
 });
 ```
+
+## Remove, Rename, and Add new Languages 
+```js
+/// Removes Common, Renames Deepspeech to Voidspeech, and adds a languages called Ochnun
+Hooks.once("setup", function addLanguages(){
+  delete CONFIG.DND5E.languages.common;
+  CONFIG.DND5E.languages.deep = "Voidspeech";
+  CONFIG.DND5E.languages.ochnun= "Ochnun";
+});
+```
+
+## Add new Item Activation Cost Types
+```js
+/// Adds in options to display in the Activation Cost dropdown
+Hooks.once("setup", function(){
+  CONFIG.DND5E.abilityActivationTypes.crithit = 'Critical Hit';
+  CONFIG.DND5E.abilityActivationTypes.attack = 'On Attack';
+  CONFIG.DND5E.abilityActivationTypes.attack = 'Replaces Attack';
+  CONFIG.DND5E.abilityActivationTypes.meleehit = 'On Melee Hit';
+  CONFIG.DND5E.abilityActivationTypes.rangedhit = 'On Ranged Hit';
+  CONFIG.DND5E.abilityActivationTypes.weaponhit = 'On Weapon Hit';
+}),
+```
+
+## Add new Weapon Types
+You will need a compendium (a [shared compendium module](https://foundryvtt.com/article/module-maker/) works well for this) in which you store a base item, just like the system already has the 'Items SRD' compendium with all the different weapons.
+Then, for each item you want to add as a new 'base weapon', copy the uuid from the item's header - right-click the book icon. Do not left-click. 
+Note that the uuid should be tweaked slightly, it should not be the full uuid but instead of this form: `module-scope.compendium-id.item-id`  
+e.g the full UUID of the Shortsword is `Compendium.dnd5e.items.Item.osLzOwQdPtrK3rQH` but in the world script only `dnd5e.items.osLzOwQdPtrK3rQH` would be used.
+
+```js
+/// Adds new Weapon types of Hand Cannon and Magnum
+Hooks.once("init", () => {
+  CONFIG.DND5E.weaponIds.handCannon = "module-scope.compendium-id.item-id";
+  CONFIG.DND5E.weaponIds.magnum = "module-scope.compendium-id.item-id";
+  // etc etc
+});
+
+```
+
+## Add new Tool Proficiencies
+Similar to the Weapon Types above, an item in a compendium will need to be referenced to create a new Tool Proficiency
+```js
+/// Adds new Tool Proficiency for Hacking Tools
+Hooks.once("init", () => {
+  CONFIG.DND5E.toolIds.hacking = "module-scope.compendium-id.item-id";

```

Comment:
```suggestion
  CONFIG.DND5E.toolIds.hacking = "<module-scope>.<compendium-id>.<item-id>";

```

Reply:
yea, I dont have super strong feelings on this, but I think it's best to leave the `<>` out. I think the explanation in the Weapon Types should help with understanding this is placeholder text and how to replace it